plugins/modules/mso_schema_template_anp_epg_selector.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Cindy Zhao (@cizhao) <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: mso_schema_template_anp_epg_selector
+short_description: Manage EPG selector in schema templates
+description:
+- Manage EPG selector in schema templates on Cisco ACI Multi-Site.
+author:
+- Cindy Zhao (@cizhao)
+version_added: '2.8'
+options:
+  schema:
+    description:
+    - The name of the schema.
+    type: str
+    required: yes
+  template:
+    description:
+    - The name of the template to change.
+    type: str
+    required: yes
+  anp:
+    description:
+    - The name of the ANP.
+    type: str
+    required: yes
+  epg:
+    description:
+    - The name of the EPG to manage.
+    type: str
+    required: yes
+  selector:
+    description:
+    - The name of the selector.
+    type: str
+  expressions:
+    description:
+    - A list of expressions associated to this selector.
+    type: list
+    suboptions:
+      expression:
+        description:
+        - An expression in expressions list
+        type: dict
+        suboptions:
+          type:
+            description:
+            - The name of the expression.
+            required: true
+            type: str
+          operator:
+            description:
+            - The operator associated to the expression.
+            required: true
+            type: str
+            choices: [notIn, in, equals, notEquals, keyExist, keyNotExist]
+          value:
+            description:
+            - The value associated to the expression.
+            type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+seealso:
+- module: mso_schema_template_anp_epg
+extends_documentation_fragment: cisco.mso.modules
+'''
+
+EXAMPLES = r'''
+- name: Add a selector to an EPG
+  cisco.mso.mso_schema_template_anp_epg_selector:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: Schema 1
+    template: Template 1
+    anp: ANP 1
+    epg: EPG 1
+    selector: selector 1
```

Comment:
Wasn't the space in the selector an issue? if yes, let's avoid using that in the examples.

Reply:
Also this example is missing the expressions.